dashboard/src/components/Sidebar.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <b-collapse id="nav-collapse">
+    <b-list-group flush>
+      <b-list-group-item
+        v-for="(item, index) in sidebar"
+        :key="index"
+        active-class="active"
+        exact
+        :to="item.link"
+      >
+        {{ item.title }}
+      </b-list-group-item>
+    </b-list-group>
+  </b-collapse>
+</template>
+
+<script>
+export default {
+  name: 'sidebar',
+  data() {
```

Comment:
Shouldn't this be `data: function() { ... }`? Or does this work also? https://vuejs.org/v2/style-guide/#Component-data-essential

Reply:
`data() {}` in an object definition is just shorthand for `data: function() {}`

Reply:
I like this. This is the same as object destructuring but with functions. I didn't know this, thanks! :100: